README.md

```diff
@@ -294,6 +294,55 @@ Openstack Object Storage settings
 - **Project ID**: Openstack project ID
 - **Container**: Name of the storage container
 
+## Applying core patches
+
+This plugin requires various trackers to be backported to maintain the plugin functionality.
+
+| Moodle version   | Patches |
+|------------------|---------|
+| Moodle 3.1       | [MDL-46375](https://tracker.moodle.org/browse/MDL-46375), [MDL-58068](https://tracker.moodle.org/browse/MDL-58068) |
```

Comment:
can you please flesh this out with version ranges like 3.1 - 3.6 so we cover everything explicitly

Reply:
Done